pulseapi/entries/forms.py

```diff
@@ -39,3 +39,11 @@ def _save_m2m(self):
         for creator_profile_id in creator_profile_id_list:
             profile_id = int(creator_profile_id)
             EntryCreator.objects.create(entry=entry, profile_id=profile_id)
+
+    def clean_thumbnail(self):
+        data = self.cleaned_data["thumbnail"]
+        if not data:
+            return data
```

Comment:
might be cleaner to say `if data and data.size > ...` on the next line, instead?